components/ai_chat/core/browser/model_validator.h

```diff
@@ -43,7 +43,9 @@ class ModelValidator {
   static bool IsValidContextSize(const std::optional<int32_t>& size);
   static bool HasValidContextSize(const mojom::CustomModelOptions& options);
 
-  static bool IsValidEndpoint(const GURL& endpoint);
+  static bool IsValidEndpoint(
+      const GURL& endpoint,
+      std::optional<bool> check_as_private_ip = std::nullopt);
```

Comment:
Hey this is a mega nit, but what do you think about calling this param `allow_private_ips`?

Reply:
Happy to leave it :smile: